201-machine-learning-advanced/azuredeploy.parameters.json

```diff
@@ -8,9 +8,6 @@
     "location": {
       "value": "westcentralus"
```

Comment:
since you're adding other clouds, this cannot be hard-coded... if the service is global, just remove this and use the defaultValue for the location param.

Reply:
Added a separate param file for us gov cloud based on [this ](https://github.com/Azure/azure-quickstart-templates/blob/57183251519684ea2c9bfdacb8522254c2f9be43/1-CONTRIBUTION-GUIDE/best-practices.md)